Deploy/pack_deeplabcut_run.spec

```diff
@@ -0,0 +1,69 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+from PyInstaller.utils.hooks import collect_all
+from PyInstaller.utils.hooks import copy_metadata
+from PyInstaller.utils.hooks import collect_dynamic_libs
+
+
+block_cipher = None
+
+datas           = []
+binaries        = []
+hiddenimports   = []
+excludes        = []
+
+# datas += copy_metadata('deeplabcut', recursive=True)
+tmp_ret = collect_all('deeplabcut')
+datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
+
+excludes = ["tensorflow","PySide6", "PyQT6", "PyQT5", "IPython", "Markdown", "jupyter", "napari", "napari_deeplabcut", "napari_console", "npe2", "napari_plugin_engine", "napari_svg", "matplotlib"]
+
+a_deeplabcut = Analysis(
+    ['../DeepLabCut/deeplabcut_run.py'],
+    pathex=['../'],
+    binaries=binaries,
```

Comment:
@kapilseh I think if we wanna exclude specific dlls, they should be passed here, you can ask Maxime to be sure